var/spack/repos/builtin/packages/flexpart/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Flexpart(MakefilePackage):
+    """FLEXPART (FLEXible PARTicle dispersion model) is a Lagrangian
+       transport and dispersion model."""
+
+    homepage = "https://www.flexpart.eu"
+    git      = "https://www.flexpart.eu/gitmob/flexpart"
+
+    maintainers = ['barche', 'tcarion']
+
+    version('10.4', tag='v10.4')
```

Comment:
Is there any tarball with a sha available for download?

Reply:
There seem to be tarballs here: https://www.flexpart.eu/downloads

I didn't use those because they don't seem to be made very consistently (the 10.4 one is not gzipped, for example).